var/spack/repos/builtin/packages/py-docutils/package.py

```diff
@@ -35,6 +35,9 @@ class PyDocutils(PythonPackage):
     depends_on("python@2.4:", when="@:0.13", type=("build", "run"))
     depends_on("py-setuptools", when="@0.15:", type="build")
 
+    # Uses 2to3
+    depends_on("py-setuptools@:57", when="@0.15", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools@:57", when="@:0.15", type="build")
```
Seems like older versions also use 2to3

Reply:
See two lines above

Reply:
Ah, that's a mistake too. 0.14 and older use distutils, but pip silently replaces all calls to distutils with setuptools. So actually, all versions depend on setuptools.

Reply:
aha, that explains why it didn't install ;p